### PR TITLE
Java doc writing

### PR DESCRIPTION
**Pull request by yasminebouamra**

### CODE REVIEW

File: src/com/usthb/modeles/Joueur.java [hunk 1]
```diff
@@ -174,7 +175,7 @@ public String getPassword() {
 	
 	/**
 	 * Donne le nombre de points du joueur depuis son inscription 
-	 * @return le score totale de la list des parties jou�es.
+	 * @return le score totale de la liste des parties jou�es.
```
Comment (abdelrahimChernai) [inline src/com/usthb/modeles/Joueur.java]: here it should be something like "le score totale **calculé à partir** de la liste des parties jouées."

File: src/com/usthb/modeles/Joueur.java [hunk 2]
```diff
@@ -116,8 +117,8 @@ public void setId() {
 	/**
 	 * <p>
 	 * Utilise la fonction de Objects pour cr�er un Num�ro s�quentiel et
-	 * utilise la fonction Math.Abs pour s'asuere que id >= 0 vu qu'il est
-	 * utilis� pour acc�der à la liste des joueurs voir les m�thodes connection
+	 * utilise la fonction Math.Abs pour s'assurer que id >= 0 vu qu'il est
+	 * utilis� pour acc�der � la liste des joueurs, voir les m�thodes connexion
```
Comment (abdelrahimChernai) [inline src/com/usthb/modeles/Joueur.java]: the method is called connection so it should be "voir les méthodes <method name> " check the exact name of the method to be sure

**Review by abdelrahimChernai — CHANGES_REQUESTED** [review]

File: src/com/usthb/modeles/Joueur.java (hunk 1 above)
Comment (yasminebouamra) [reply src/com/usthb/modeles/Joueur.java]: c'est corrigé

File: src/com/usthb/modeles/Joueur.java (hunk 2 above)
Comment (yasminebouamra) [reply src/com/usthb/modeles/Joueur.java]: corrected

**Review by abdelrahimChernai — APPROVED** [review]

**Pull request merged**